### DISCUSSION TITLE
Quality Assurance of Metadata

### DISCUSSION THREAD

**Discussion by AndersOstmanNovogit** [gopeg/issues-barriers#45] [General]
Many portals and geoportals are provided by public agencies. There is an expectations that information provided by public agencies are correct and up-to-date. This require that efficient QA routines for metadata management are embedded in the operations of the portals. This may also have the consequence that some groups of metadata providers are excluded from the portals. In contrast to this, the Swedish data portal stress that it is the owner of the data sets that is responsible for its quality. In fact, one general recommendation is that it is better to publish data with poor quality than to wait for publication until the data sets gets better.
On the other hand, the publishing of metadata is often hampered by the extra amount of work required to create and maintain the metadata, so just adding more responsibilities to the metadata providers does not seem to be a viable solution. The amount of work is more pronounced in portals based on the ISO 19115 metadata standard.
In cases where metadata validation procedures are applied, the extra work to correct for the errors being identified may be seen as an additional barrier. In addition, the costs for sharing data often occurs at one institution while the benefits occurs at other institutions. This requires a cooperation at a high level.
The german geoportal state that "since there is no negative feedback from data users about the sometimes poor quality of the metadata, a high level of effort is sometimes seen compared to a low benefit in metadata maintenance. One way to improve this situation would be to automate the creation and updating of metadata (as far as this makes sense and is possible). That would reduce costs and improve quality. The generation of metadata should take place in a single-stage process together with the provision of the data".
In  Italy, possible barriers refer to limited human and/or financial resources of some public administrations (for instance small municipalities) that don’t enable them, e.g, to implement a local catalogue or infrastructure. This barrier could be addressed with a subsidiarity mechanism by administrations with greater capabilities.

**Comment by thorsten-reitz** [comment]
The way that we dealt with this barrier is by automating metadata generation as well as publication into different formats/targets. This is also what was done to bring the 130k+ planned land use data sets online with metadata. The work is then limited to set up the generation process once.
The automation also allows us to update the metadata automatically when the validation rules are changed, and when the data or any information that goes into the metadata, such as contact info, changes.

**Reply by AndersOstmanNovogit** [reply]
Some questions:
Line 1: I assume you mean that a new metadata record is generated for each different data file being produced, for instance one record for a gpkg file and another record for the gml file covering the same area.
Line 2: Were the data sets planned or were the data set about land use planning?

**Comment by AndersOstmanNovogit** [comment]
This issue is reported in section 4.5.2 in D3.4
An amendment to the solution section is as follows

The MQA tool can be used for evaluating the quality of metadata in a standardised way. However, this tool is only to be used on already published metadata records. There are also several other tools having similar functionality, see for instance https://fairassist.org/#!/. One should however note that these tools evaluate the quality of published metadata. These Quality Evaluation procedures are different from Quality Assurance procedures, which aims at ensuring that certain metadata records are of sufficient quality. QA procedures are often implemented as acceptance tests, relying on some kind of evaluation. One key question when designing Q routines are the effectiveness and efficiency of the acceptance tests and to which degree they contribute to the overall quality. 

Preferred solution: In the GO-PEG project, the MQA tool has been used to evaluate the quality of the metadata published in the European Data Portal. Experiments have been made in using the FAIRenough tool in acceptance testing earlier in the publishing process. In addition, structured checklists have been considered to be used very early in the publishing process.

**Reply by AndersOstmanNovogit** [reply]
Now reported in D4.3 section 4.4.3
